ui/src/kapacitor/containers/KapacitorRulePage.js

```diff
@@ -1,31 +1,32 @@
 import React, {PropTypes, Component} from 'react'
 import {connect} from 'react-redux'
-import _ from 'lodash'
 
 import * as kapacitorRuleActionCreators from 'src/kapacitor/actions/view'
 import * as kapacitorQueryConfigActionCreators from 'src/kapacitor/actions/queryConfigs'
 
 import {bindActionCreators} from 'redux'
 import {getActiveKapacitor, getKapacitorConfig} from 'shared/apis/index'
-import {RULE_ALERT_OPTIONS, DEFAULT_RULE_ID} from 'src/kapacitor/constants'
+import {DEFAULT_RULE_ID} from 'src/kapacitor/constants'
 import KapacitorRule from 'src/kapacitor/components/KapacitorRule'
+import parseHandlersFromConfig from 'src/shared/parsing/parseHandlersFromConfig'
 
 class KapacitorRulePage extends Component {
   constructor(props) {
     super(props)
 
     this.state = {
-      enabledAlerts: [],
+      handlersFromConfig: [],
       kapacitor: {},
     }
   }
 
   async componentDidMount() {
     const {params, source, ruleActions, addFlashMessage} = this.props
-    if (this.isEditing()) {
-      ruleActions.fetchRule(source, params.ruleID)
-    } else {
+
+    if (params.ruleID === 'new') {
```

Comment:
just in case, here's another instance of `'new'`

Reply:
thanks, would you prefer to pair on this?

Reply:
nope, just making sure that these are accurate since this string has been the root of two issues that i've reported in this PR